### ISSUE TITLE
Left bracket in string value breaks pretty view

### ISSUE THREAD

**Issue by Graham42** [a85/POSTMan-Chrome-Extension#1152]
Chrome Version: Version 43.0.2357.125 (64-bit)
Postman Version: 3.0.0.6 (packaged)
OS: linux (Fedora)

The pretty view breaks if any string value has brackets in it. The following api return gives an error: `Malformed JSON: Unexpected 'I'`

``` json
)]}',
{
    "result":{
        "@type":"somemetavalue",
        "@id":"http://host.name/some/meta/id",
        "description":"A bunch of text that and (IT) and then more text."
    }
}
```

**Comment by abhijitkane** [comment]
@Graham42 can you confirm this fixed in v3.0.4?

**Comment by Graham42** [comment]
Confirming fixed
